npm/npm.go

```diff
@@ -58,20 +60,24 @@ type NetworkPolicyManager struct {
 // NewNetworkPolicyManager creates a NetworkPolicyManager
 func NewNetworkPolicyManager(config npmconfig.Config,
 	informerFactory informers.SharedInformerFactory,
+	podFactory informers.SharedInformerFactory,
 	dp dataplane.GenericDataplane,
 	exec utilexec.Interface,
 	npmVersion string,
-	k8sServerVersion *version.Info) *NetworkPolicyManager {
+	k8sServerVersion *version.Info,
+	npmLiteToggle bool) *NetworkPolicyManager {
 	klog.Infof("API server version: %+v AI metadata %+v", k8sServerVersion, aiMetadata)
 
 	npMgr := &NetworkPolicyManager{
-		config:    config,
-		Dataplane: dp,
+		config:        config,
+		Dataplane:     dp,
+		NpmLiteToggle: npmLiteToggle,
```

Comment:
set this with config.Toggles.NPMlite ? instead of a new bool

Reply:
yes changed it, thank you for pointing that out